### ISSUE TITLE
Error end animation

### ISSUE THREAD

**Issue by KingOfDragons** [kimmobrunfeldt/progressbar.js#107]
When i hide de progress bar and reste to 0 it show me this in the log!!

```java
TypeError: this._finish is undefined
	

this._finish.call(this, this._currentState, this._attachment);

	
```

**Comment by kimmobrunfeldt** [comment]
Could you create a reproducable example in JSFiddle?

**Comment by Ishra** [comment]
I don't have a fiddle for you but i have tracked the down the error.

on row 221:
step(targetState, tweenable._attachment, timeoutHandler_offset);
tweenable.stop(true);

if stop is called from within the step during the last iteration the error will appear when tweenable.stop(true) is called.

**Comment by kimmobrunfeldt** [comment]
Thanks for reporting this. It occurs because we are calling .dispose() method of the underlying shifty Tweenable here: https://github.com/kimmobrunfeldt/progressbar.js/blob/df29df4211620e99143cb47120482760a0e508ce/src/path.js#L155

The dispose shouldn't be necessary as we are letting go of the tweenable reference. If I confirmation to this from shifty author in this issue https://github.com/jeremyckahn/shifty/issues/86, I can safely remove the dispose call and it will fix this issue.

**Comment by kimmobrunfeldt** [comment]
Fix to this released in 1.0.0 https://github.com/kimmobrunfeldt/progressbar.js/releases/tag/1.0.0
